backend/app/models/account.rb

```diff
@@ -25,7 +26,7 @@ class Account < ApplicationRecord
   validates :twitter, url: true
   validates :facebook, url: true
   validates :instagram, url: true
-  validates :language, inclusion: {in: Language::TYPES}
+  validates :language, inclusion: {in: Language::TYPES, allow_blank: true}, presence: true
```

Comment:
just question, what is reason behind this? I see that you have not touched validation tests so it behaves same way as before so not sure why this change was required